juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
402
/home/jenkins/.jenkins/workspace/iceberg/jfit-rules_validate/rules/juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml
  5:9       error    wrong indentation: expected 4 but found 8  (indentation)
  5:20      warning  truthy value is not quoted  (truthy)
  8:9       error    wrong indentation: expected 4 but found 8  (indentation)
  12:1      error    trailing spaces  (trailing-spaces)
  15:16     error    too many spaces after colon  (colons)
  16:16     error    too many spaces after colon  (colons)
  17:16     error    too many spaces after colon  (colons)

Reply:
Addressed and resolved